lib/ui/widgets/custom_list_tile.dart

```diff
@@ -4,13 +4,20 @@ class CustomListTile extends StatelessWidget {
   final IconData iconData;
   final String text;
 
-  CustomListTile({this.iconData,this.text});
+  CustomListTile({this.iconData, this.text});
```

Comment:
What is this for ?

Reply:
this is been used in SideMenu which is not created by me